Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -112,22 +112,36 @@ configuration:
   required: false
   type: 15
   section: Collect
-- additionalinfo: When selected, closing the Splunk notable event is mirrored in Cortex XSOAR.
+- additionalinfo: If selected, closing a Notable Event on Splunk with a status of "Closed" will close the corresponding fetched incident on XSOAR.
   defaultvalue: 'false'
-  display: Close Mirrored XSOAR Incident
+  display: Mirror closed XSOAR Incidents (Incoming Mirroring)
   name: close_incident
   required: false
   type: 8
   section: Collect
   advanced: true
-- additionalinfo: When selected, closing the Cortex XSOAR incident is mirrored in Splunk.
+- additionalinfo: If selected, Notables with a status label that is marked as "End Status" on Splunk will also be mirrored on closure.
```

Comment:
If selected, Notables with a status label that are marked as "End Status" in Splunk will also be mirrored on closure.